### ISSUE TITLE
Key map dialog appearing outside of window bounds

### ISSUE THREAD

**Issue by Zetaphor** [UltimateHackingKeyboard/agent#2424]
Running on Windows 11 until I figure out getting Agent working on NixOS

This is on a 4K display:

![image](https://github.com/user-attachments/assets/9c02c875-eb82-4a7f-a503-3f8d4c5cc0fe)

**Comment by ert78gb** [comment]
@mondalaci I can't reproduce it on my windows 10 notebook. It does not have 4K display.

@Zetaphor Do you still experience the wrong positioning of the dialog with the latest Agent version?

**Comment by mondalaci** [comment]
@Zetaphor Nobody else has reported this issue, and I can't reproduce it on Linux Mint, so I'm suspicious of a non-standard configuration / settings. The [high-]DPI settings used or any related settings would be welcome.

**Comment by pcooke9** [comment]
I use 4k resolution with Win11 display scaling set to 150%.

I've seen similar issues manifesting in various ways for a long time. I can't currently reproduce it exactly the same as the image in the first post, but there are some glitches.

While I was trying to reproduce it just now (with recent Agent master branch), I discovered a funny little quirk. The outer edges of the keys teleport to the opposite side of the keymap when resizing the window.

This video was taken with Win 11 at 4k res and 150% display scaling, but I can reproduce it with 100% scaling as well:
https://www.youtube.com/watch?v=JuQUrXqIrQY

**Comment by mondalaci** [comment]
Thanks for sharing! The teleporting issue only seems to happen when the application window is very small. I don't think users use such a small window size, so we should only address this if the fix is very easy.

**Comment by pcooke9** [comment]
Of course, I can't imagine anyone actually trying to use Agent with the window that small.😆 I just thought it was interesting, and posted it in case it could potentially help with something down the road.

**Comment by ert78gb** [comment]
The main issue solved in the #2610 so I close it
